src/screens/TopicPageScreen/index.js

```diff
@@ -396,10 +401,11 @@ const TopicPageScreen = (props) => {
     />
   );
 
-  if (isInitialLoading) return null;
   return (
-    <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
+    <View>
       <NavHeader
+        isLoading={isInitialLoading}
+        opacityImage={opacityImage}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `NavHeader` component now receives two new props: `isLoading` and `opacityImage`. It's important to ensure that these props are handled correctly within the `NavHeader` component. Also, the `View` component has replaced the `SafeAreaProvider` as the root container. This change might affect the layout on devices with notches or rounded corners. 



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
it's expected ser, we want to remove the safe area